src/components/complibrary/clearcache.jsx

```diff
@@ -0,0 +1,13 @@
+import React, {Component} from 'react';
+
+class ClearCache extends Component {
```

Comment:
this can easily be a `PureComponent` https://reactjs.org/docs/react-api.html#reactpurecomponent